<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ứng Dụng Kiểm Tra</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_welcome">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Chào mừng!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_user_info">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>300</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Người dùng: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_highest_score">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Điểm cao nhất: 0</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_question">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>560</width>
      <height>300</height>
     </rect>
    </property>
    <property name="title">
     <string>Câu hỏi</string>
    </property>
    <widget class="QLabel" name="label_question_number">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Câu 1/5:</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_question">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>520</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Câu hỏi sẽ hiển thị ở đây...</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_option1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>520</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Đáp án A</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_option2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>520</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Đáp án B</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_option3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>520</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Đáp án C</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_option4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>520</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Đáp án D</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_next">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>250</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Tiếp theo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_previous">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>250</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Trước</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_submit">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>250</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Nộp bài</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_start_test">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>420</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Bắt đầu kiểm tra</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_logout">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Đăng xuất</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui> 